src/unix/linux-core.c

```diff
@@ -983,3 +983,37 @@ void uv__set_process_title(const char* title) {
   prctl(PR_SET_NAME, title);  /* Only copies first 16 characters. */
 #endif
 }
+
+int uv__enable_largesocket(uv_stream_t *stream) {
+  int rmem = 0;
+  int wmem = 0;
+  int rs = 0;
+  int ws = 0;
+  int rr = -1;
+  int rw = -1;
+  FILE *fp = uv__open_file("/proc/sys/net/core/rmem_max");
+  if (fp == NULL)
+    return UV_ENOSYS;
+  if (fscanf(fp, "%d", &rmem) == 1)
+    rs = 1;
+  fclose(fp);
+
```

Comment:
done - I guess you meant to return in either case - so using OR instead of AND.
`rs != 1 || rmem <= (64 * 1024)`